lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -113,13 +113,9 @@ function convertMessageToCtc(message, examples = {}) {
 
   // Process each placeholder type
   _processPlaceholderMarkdownCode(icuDefn);
-
   _processPlaceholderMarkdownLink(icuDefn);
-
   _processPlaceholderCustomFormattedIcu(icuDefn);
-
   _processPlaceholderDirectIcu(icuDefn, examples);
-
```

Comment:
Whitespace like this come down to author preference and should really be discussed in the original PR rather than changed in the next commit (inviting style edit wars). Presumably @exterkamp preferred it and no one brought it up in the review, so unless there's a reason to change it, the default move should be to leave it unchanged.

Reply:
ok sorry bud